wagtail_wordpress_import/prefilters/transform_styles_filter.py

```diff
@@ -135,19 +221,45 @@ def transform_tag_em(soup, tag):
         ),
         transform_style_center,
     ),
-    (re.compile(r"float:left*", re.IGNORECASE), transform_style_left),
-    (re.compile(r"float:right*", re.IGNORECASE), transform_style_right),
+    (re.compile(r"text-align:left*", re.IGNORECASE), transform_style_left),
+    (re.compile(r"text-align:right*", re.IGNORECASE), transform_style_right),
+    (re.compile(r"float:left*", re.IGNORECASE), transform_float_left),
+    (re.compile(r"float:right*", re.IGNORECASE), transform_float_right),
 ]
```

Comment:
**Suggestion**: It feels like these default configurations should be somewhere in a separate (`defaults.py`?) module, and probably we'd need to put the individual transformation functions either there alongside or in their own module to prevent circular imports. Does that sound reasonable?